packages/loader/container-definitions/src/runtime.ts

```diff
@@ -135,6 +137,7 @@ export interface IContainerContext extends IDisposable {
     readonly serviceConfiguration: IClientConfiguration | undefined;
     readonly version: string;
     readonly previousRuntimeState: IRuntimeState;
+    pendingOps?: unknown;
```

Comment:
It might be useful not to use "ops" terminology at interface level, as IRuntime should be free to use whatever serialization mechanism it wants.